src/main/java/app/controllers/UserController.java

```diff
@@ -34,7 +34,7 @@ private static void createUser(Context ctx, ConnectionPool connectionPool)
                 UserMapper.createuser(username, password1, connectionPool);
                 ctx.attribute("message", "Du er hermed oprettet med brugernavn: " + username +
                         ". Nu skal du logge på.");
-                ctx.render("index.html");
+                ctx.render("startPage.html");
```

Comment:
please keep using the index as it will otherwise break the application for all other teams

Reply:
renamed to index.html